resfulservice/misc/README.md

```diff
@@ -0,0 +1,8 @@
+On the terminal:
+1. Go to => `~/.bash_profile` (If not found create one)
+2. Scroll down to the end of the `.bash_profile` file
+3. Download gpg [here](https://gpgtools.org/) and request decryption keys for `envVarFile.gpg`
+4. Decrypt `envVarFile.gpg` content by executing `decryptENVARFile.sh` in this directory
+5. Copy and paste `envVarFile` content
```

Comment:
It may make more sense to the reader to list these three steps before creating / opening up .bash_profile

Reply:
@stouffers, thanks for this suggestion. I'll update.